### ISSUE TITLE
Siteimprove Issue: Naming generic landmarks

### ISSUE THREAD

**Issue by brharp** [ccswbs/hjckrrh#791]
Reported by SiteImprove.

SiteImprove says:
> The page contains an HTML5 `<section>` or a WAI-ARIA landmark of the type 'region'. These types of
> landmarks are very general and should be named to help users identify the purpose of the landmark.
> Use either the WAI-ARIA attribue 'aria-label' or 'aria-labelledby' to name the landmark in a way 
> that clearly explains the purpose of the landmark to all users.

They go on to say:
> In order to sufficiently meet this criterion, one of the following techniques can be applied.
> 1. [ARIA20](http://www.w3.org/TR/WCAG20-TECHS/ARIA20) Using the region role to identify a region of the page
>
> The references are taken from the official accessibility guidelines WCAG 2.0.

The ARIA20 reference describes how to label elements with `role="region"`. We are not using `role="region"`, but we are using `<section>` elements which [default](https://fae.disability.illinois.edu/rulesets/ROLE_5/) to 'region' if no 'role' attribute is specified.

This error is reported on every page.
